code/src/layouts/intro-banner/IntroBanner.tsx

```diff
@@ -0,0 +1,54 @@
+import AppTypography from "@/components/app-typography/AppTypography";
+import AppContainer from "@/components/app-container/AppContainer";
+import AppButton from "@/components/app-button/AppButton";
+import AppBox from "@/components/app-box/AppBox";
+import AppBadge from "@/components/app-badge/AppBadge";
+
+import { introBannerImages } from "@/layouts/intro-banner/IntroBanner.constants";
+import useIntroBanner from "@/layouts/intro-banner/useIntroBanner";
+
+import "@/layouts/intro-banner/IntroBanner.scss";
+
+const IntroBanner = () => {
+  const currentImageIndex = useIntroBanner(introBannerImages);
+
+  return (
+    <AppBox
+      className="spa-banner-intro"
+      style={{
+        backgroundImage: `url(${introBannerImages[currentImageIndex]})`
+      }}
+    >
+      <AppContainer maxWidth="lg">
+        <AppBox className="spa-banner-intro__wrapper">
+          <AppBadge
+            color="danger"
```

Comment:
Why `color`? Maybe use `variant` instead?

Reply:
Yeah, I forgot about it. Thank you, fixed